project/CodeCoverageSettings.scala

```diff
@@ -13,7 +13,7 @@ object CodeCoverageSettings {
 
   val settings: Seq[Setting[?]] =   Seq(
     ScoverageKeys.coverageExcludedPackages := excludedPackages.mkString(";"),
-    ScoverageKeys.coverageMinimumStmtTotal := 94,
+    ScoverageKeys.coverageMinimumStmtTotal := 93,
```

Comment:
Rather than lower this, I'd try to figure out whether we need that newly introduced file (which is probably is why the coverage has dropped).

Reply:
It's been implemented because we are no more using the AuthRedirects that was in the older play version. Do you suggest to put all of the necessary bits of code in already existing StrideAuth.scala?

Reply:
OK, verified that it was copied and trimmed from the old version of the library. I'd recommend also copying or adding the tests for this class.